app/stac_api/models.py

```diff
@@ -454,6 +473,7 @@ def upload_asset_to_path_hook(instance, filename=None):
     # set the hash to the storage to use it for upload signing, this temporary attribute is
     # then used by storages.S3Storage to set the MetaData.sha256
     setattr(instance.file.storage, '_tmp_sha256', ctx.hexdigest())
+    setattr(instance.file.storage, '_tmp_update_interval', instance.update_interval)
```

Comment:
Does the above comment need to be updated, so that `_tmp_update_interval` appears in the comment, too?